### ISSUE TITLE
Dependabot: Insufficient Granularity of Access Control in JSDom

### ISSUE THREAD

**Issue by MaeganWood64** [US-EPA-CAMD/PI-1#3755]
As of 8/24, no available change 

Present in 

- [ ] easey-ecmps-ui

- [ ] easey-campd-ui

**Comment by MaeganWood64** [comment]
- Something to look into post Beta Release, as the fix is contingent on updating react scripts which would cause several breaking changes

**Comment by MaeganWood64** [comment]
Ticket no longer needed
